### DISCUSSION TITLE
关于GKD图标背景颜色的疑问

### DISCUSSION THREAD

**Discussion by AIsouler** [gkd-kit/gkd#703] [General]
在开启系统深色模式的情况下安装gkd，图标背景色是白色的，关闭系统深色模式的情况下安装gkd，图标背景色是黑色的，请问是这样设计的还是系统特性？

https://github.com/user-attachments/assets/9a66cdf3-43b2-4ce3-a721-7ba0e05547dc

**Comment by AIsouler** [comment]
貌似是第三方安装器的问题

**Comment by lisonge** [comment]
图标适配了深色浅色主题，不同主题显示不同图标

参考代码

https://github.com/android/nowinandroid/blob/main/app/src/main/res/drawable/ic_launcher_foreground.xml#L24-L24

至于图标为什么不在切换深色主题后变换，应该是国产系统不支持

手动切换也很简单，到手机主题切换一个图标包，然后切回来，就会显示当前主题正确的图标颜色
